### PR TITLE
Footer section

### PR DESCRIPTION
**Pull request by carlows**
# Context

This PR implements the footer section, including the CTA section. 

The CTA section is optional in some pages (like the 404 page), so I added a parameter to the footer include that can be used to show/hide that section: 

```
{% include footer.html include_cta=true %}
```

The footer is sticky, so it's always at the bottom no matter what the content is.

## Media

image

image

image

With `include-cta=false`:

image

### CODE REVIEW

File: _sass/_components--button.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.btn-lg {
+  padding: $spacer*1.25 $spacer*3.75;
```
Comment (duranmla) [inline _sass/_components--button.scss]: There is "$btn-padding-y-lg, $btn-padding-x-lg, $font-size-lg, $btn-border-radius-lg" variables for btn-lg

Comment (TheMightyPenguin) [reply _sass/_components--button.scss]: Also @duranmla, buttons should be an object or component?

File: _sass/_tools--spacing.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.outer-section-padding {
+  padding-top: $spacer*3.75;
```
Comment (duranmla) [inline _sass/_tools--spacing.scss]: You can use `@include padding-y($spacer*3.75)`

**Review by duranmla — CHANGES_REQUESTED** [review]
Few comments and then LGTM

File: _includes/footer-cta.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<div class="footer-cta outer-section-padding">
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-lg-6">
+        <div class="brand-list text-center text-lg-left">
```
Comment (TheMightyPenguin) [inline _includes/footer-cta.html]: can we make brand-list an include with parameters? Well, it's not used in other parts of the site tho, but still would be nice to have

File: _sass/_partials--footer.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+.footer-links {
+  color: $color--font-tertiary;
+}
+
+.footer-link-list {
+  list-style: none;
+  font-weight: normal;
```
Comment (TheMightyPenguin) [inline _sass/_partials--footer.scss]: keep alphabetical ordering on css properties

**Review by TheMightyPenguin — CHANGES_REQUESTED** [review]

File: _sass/_partials--footer.scss [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  li a {
+    color: $color--font-tertiary;
+  }
+}
+
+.section-separator {
+  height: 2px;
+  background-image: radial-gradient(circle at 50% 0, #303030, #000000);
```
Comment (TheMightyPenguin) [inline _sass/_partials--footer.scss]: same here for alphabetical ordering

File: _sass/_tools--spacing.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+.outer-section-padding {
+  @include padding-y($spacer*3.75);
```
Comment (TheMightyPenguin) [inline _sass/_tools--spacing.scss:2]: 👌  this is gonna be useful for section spacing

**Review by TheMightyPenguin — APPROVED** [review]
LGTM

**Review by duranmla — APPROVED** [review]

**Pull request merged**